packages/block-library/src/site-title/edit/index.js

```diff
@@ -1,37 +1,58 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
 /**
  * WordPress dependencies
  */
 import { useEntityProp } from '@wordpress/core-data';
-import { BlockControls, RichText } from '@wordpress/block-editor';
 import { __ } from '@wordpress/i18n';
+import {
+	RichText,
+	AlignmentToolbar,
+	BlockControls,
+	__experimentalBlock as Block,
+} from '@wordpress/block-editor';
 
 /**
  * Internal dependencies
  */
 import LevelToolbar from './level-toolbar';
 
-export default function SiteTitleEdit( {
-	attributes: { level },
-	setAttributes,
-} ) {
+export default function SiteTitleEdit( { attributes, setAttributes } ) {
+	const { level, textAlign } = attributes;
 	const [ title, setTitle ] = useEntityProp( 'root', 'site', 'title' );
-	const tagName = level === 0 ? 'p' : `h${ level }`;
+	const tagName = 0 === level ? 'p' : `h${ level }`;
```

Comment:
We don't enforce Yoda conditions in our JS (since we're enforcing typed comparisons -- `===` instead of `==` -- anyway).

Reply:
Actually I copy-pasted it from the previous source and yeah, it felt strange to have Yoda conditions in JS. I'll revert this change since I don't like it either.